node/cn/state_accessor.go

```diff
@@ -214,6 +214,9 @@ func (cn *CN) stateAtTransaction(block *types.Block, txIndex int, reexec uint64)
 	if err != nil {
 		return nil, vm.BlockContext{}, vm.TxContext{}, nil, nil, err
 	}
+	// If prague hardfork, insert parent block hash in the state as per EIP-2935.
+	cn.engine.Initialize(cn.blockchain, block.Header(), statedb)
```

Comment:
I think this should be removed as well, just like `Initialize` is not called from `cn.stateAtBlock`.

Reply:
Sorry, I was mistaken. Never mind